esper_group_actions.py

```diff
@@ -252,11 +254,30 @@ def get_devices_in_group(group_id):
     except ApiException as api_exception:
         print("Exception when calling DeviceApi->get_all_devices: %s\n" % api_exception)
 
+def get_all_devices_in_enterprise():
+    # create an instance of the API class
+    api_instance = esperclient.DeviceGroupApi(esperclient.ApiClient(CONFIGURATION))
+    try:
+        api_response = api_instance.get_all_groups(ENTERPRISE_ID)
+        if len(api_response.results):
+            for group in api_response.results:
+                #print(group.id)
+                # add all the devices in this group to global list of devices
+                get_devices_in_group(group.id)
```

Comment:
I believe you will have to ensure you go through all "pages" of results one-by-one to acquire all of them. This will give you the first 20, if I am not mistaken.

Reply:
Updated limit to 5000. 
No of groups should not go more than.